tools/resgen/src/main.cpp

```diff
@@ -32,15 +35,22 @@
 using namespace std;
```

Comment:
I think the previous way of prefixing with `std::` was actually better.  Even though there was an unnecessary `using namespace std`.  Maybe you can ask AI to undo just that part?

Reply:
yeah, lol, I was thinking the same thing. Let me try!

Reply:
done